ui/app/components/app/account-menu/account-menu.component.js

```diff
@@ -94,6 +96,8 @@ export default class AccountMenu extends PureComponent {
         return kr.accounts.includes(simpleAddress) || kr.accounts.includes(identity.address)
       })
 
+      const addressIconArray = addressConnectedIconMap[identity.address]
```

Comment:
I believe this was only meant to include the icon for the active tab (if it's connected), rather than all connected sites.